lms/templates/courseware/progress.html

```diff
@@ -12,6 +12,7 @@
 <%block name="headextra">
 <%static:css group='style-course-vendor'/>
 <%static:css group='style-course'/>
+<script type="text/javascript" src="${static.url('js/vendor/backbone-min.js')}"></script>
```

Comment:
Again, this should not be needed (and would result in the file being unnecessarily downloaded twice).